### ISSUE TITLE
can change menu item layout?

### ISSUE THREAD

**Issue by siavash-sajjad** [SpecialCyCi/AndroidResideMenu#152]
hi
i wanna change text color and remove image from menu item. is it possible?
i create new layout and create new class with extends from ResideMenuItem but no't show text

**Comment by akardas16** [comment]
yes you can.  find external libraries in android stüdio and find  residemenu library after that you can see res folder and you can change color of items or do other things change paddings margins...
